tests/integration/targets/docker_plugin/aliases

```diff
@@ -0,0 +1,2 @@
+shippable/posix/group1
```

Comment:
```suggestion
shippable/posix/group4
```
Most 'plain' docker tests should go into group 4, which tests in more environments than groups 1-3 (used for swarm tests). (Group 5 is reserved for the docker_container tests.)

Reply:
Thanks, I was confused with the groups. Is there any doc with such information given?